app/templating/summary/formatters/check_box.py

```diff
@@ -0,0 +1,13 @@
+from app.templating.summary.formatters.abstract_formater import AbstractFormater
+
+
+class CheckBoxFormatter(AbstractFormater):
+
+    @staticmethod
+    def format(schema, state, user_answer):
+        answers = []
+        for option in schema.answers[0].options:
```

Comment:
We use schema in schema.answers[0].options, but we can tighten up as suggested on the other comments to just pass schema.answers.